ricktaylorHX/ricktaylorHX.github.io
# index.html tidy up and site background colour change

#### What does this PR do? (please provide any background)
Tidies up index.html to only display one picture and have one paragraph of text.

#### What tests does this PR have?
N/A

#### How can this be tested?
Check index.html to see if single image and paragraph display correct. Check some other pages on the site to check background colour is as expected (white)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Review:
- [x] I've tested and checked the work, and am happy for it the branch to be merged.
